README.md

```diff
@@ -1,54 +1,69 @@
+<picture align="center">
+  <img src="https://user-images.githubusercontent.com/45081667/227194559-57c73a9e-51df-4d42-b9f0-8b6dd011ae7b.png" alt="Header" >
+</picture>
```

Comment:
Hmm, I think I know what you mean, but that is built inside the picture which I did not touch because is marketing material.  The picture itself should be entered due to `align=center` directive.